corehq/apps/repeaters/dbaccessors.py

```diff
@@ -70,23 +70,23 @@ def get_paged_repeat_records(domain, skip, limit, repeater_id=None, state=None):
         'reduce': False,
         'limit': limit,
         'skip': skip,
+        'descending': True,
     }
 
     if repeater_id and not state:
-        kwargs['startkey'] = [domain, repeater_id]
-        kwargs['endkey'] = [domain, repeater_id, {}]
+        kwargs['endkey'] = [domain, repeater_id]
+        kwargs['startkey'] = [domain, repeater_id, {}]
```

Comment:
good question, so this has to do with how couch treats `descending=True`. this changes the order of the index in couch:

> View rows are sorted by the key; specifying descending=true will reverse their order. Note that the descending option is applied before any key filtering, so you may need to swap the values of the startkey and endkey options to get the expected results. The sorting itself is described in ViewCollation.

https://wiki.apache.org/couchdb/HTTP_view_API

The reason you don't see me switching the endkey and start key on line 36 is that i don't use descending=True since i'm just getting the count (it's a different function).

As for why i'm now using an empty dict, i think it's best understood if i use an example. say you have an index that looks like this:

```
['domain1', 'repeater-1', 'PENDING', '2017']
['domain1', 'repeater-1', 'PENDING', '2018']
['domain1', 'repeater-2', 'FAIL', '2017']
['domain1', 'repeater-2', 'PENDING', '2017']
```
those are ordered the way couch would order the keys in an index (repeater-2 is string comparison greater than repeater-1). when i say i want `startkey=['domain1', 'repeater-1', 'PENDING']` and `endkey=['domain1', 'repeater-1', 'PENDING',{}]`, it means i want everything in between those two keys. `{}` is a way of specifying the greatest value in a string comparison. since everything is less than `{}` it will essentially ignore the timestamp param, which is what i want. i'm only using the timestamp param to have couch order the results. does that make sense? happy to offline